cardshifter-core/src/main/java/com/cardshifter/core/groovy/WhilePresentDelegate.groovy

```diff
@@ -50,7 +50,15 @@ class WhilePresentDelegate {
             [withPriority: {int priority ->
                 [on: {Closure filter ->
                     addModifier(resources, priority, filter, {deleg ->
-                        'Give ' + deleg.description + ' ' + change + ' ' + resources.join(' and ')
+                        String changeStr
+                        if (change == 0) {
+                            changeStr = 'unchanged'
+                        }
+                        else {
+                            changeStr = [change.abs(), change < 0 ? 'less' : 'more'].join(' ')
+                        }
+
+                        [deleg.description.capitalize(), 'have', changeStr, resources.join(' and ')].join(' ')
```

Comment:
This line and line 58 are a bit hard to understand (line 58 was very clever btw and slightly easier to read). Comments would help, perhaps with an example of the end result?